__tests__/app.test.js

```diff
@@ -70,6 +70,128 @@ describe("/api/reviews", () => {
+    return request(app)
+      .get("/api/reviews?sort_by=designer")
+      .expect(200)
+      .then((res) => {
+        expect(Array.isArray(res.body)).toBe(true);
+        expect(res.body).toHaveLength(13);
+        res.body.forEach((review) => {
+          expect(review).toEqual(
+            expect.objectContaining({
+              review_id: expect.any(Number),
+              title: expect.any(String),
+              comment_count: expect.any(String),
+              designer: expect.any(String),
+              review_img_url: expect.any(String),
+              votes: expect.any(Number),
+              category: expect.any(String),
+              owner: expect.any(String),
+              created_at: expect.any(String),
+            })
+          );
+        });
+        expect(res.body).toBeSortedBy("designer", {
+          descending: true,
+        });
+      });
+  });
+  test("GET - 200 responds with an array of review objects sorted by date ascending", () => {
+    return request(app)
+      .get("/api/reviews?order=ASC")
+      .expect(200)
+      .then((res) => {
+        expect(Array.isArray(res.body)).toBe(true);
+        expect(res.body).toHaveLength(13);
+        res.body.forEach((review) => {
+          expect(review).toEqual(
+            expect.objectContaining({
+              review_id: expect.any(Number),
+              title: expect.any(String),
+              comment_count: expect.any(String),
+              designer: expect.any(String),
+              review_img_url: expect.any(String),
+              votes: expect.any(Number),
+              category: expect.any(String),
+              owner: expect.any(String),
+              created_at: expect.any(String),
+            })
+          );
+        });
+        expect(res.body).toBeSortedBy("created_at", {
+          ascending: true,
+        });
+      });
+  });
+  test("GET - 200 can handle multiple queries", () => {
+    return request(app)
+      .get(
+        "/api/reviews?category=social%20deduction&sort_by=designer&order=ASC"
+      )
+      .expect(200)
+      .then((res) => {
+        expect(Array.isArray(res.body)).toBe(true);
+        expect(res.body).toHaveLength(11);
+        res.body.forEach((review) => {
+          expect(review).toEqual(
+            expect.objectContaining({
+              review_id: expect.any(Number),
+              title: expect.any(String),
+              comment_count: expect.any(String),
+              designer: expect.any(String),
+              review_img_url: expect.any(String),
+              votes: expect.any(Number),
+              category: "social deduction",
+              owner: expect.any(String),
+              created_at: expect.any(String),
+            })
+          );
+        });
+        expect(res.body).toBeSortedBy("designer", {
+          ascending: true,
+        });
+      });
+  });
+  test("GET - 200 an invalid category returns an empty array", () => {
```

Comment:
Be careful - we have two cases here. A valid category in the database with no associated reviews should return an empty array and 200. What about something that isn't in the database altogether?